### DISCUSSION TITLE
cg_bonds visualization error

### DISCUSSION THREAD

**Discussion by CongL415** [Martini-Force-Field-Initiative/MartiniDiscussions#11] [Q&A]
Dear developer, 

I am trying to visualize elastic bonds of a multi-domain protein system. The system was prepared using charmm-gui and the simulation went well. However, when I tried to visualize the elastic bonds using the following command:
cg_bonds -top system.top -topoltype "elastic"

I got an error from vmd saying that "atomsel: setbonds: Need one bondlist for each selected atom". I tried both the latest cg_bonds.tcl and cg_bonds-v5.tcl and none of them worked. I also attached the system I am working with. Could you help me figure it out?

Best,
Cong
[cg_system.zip](https://github.com/user-attachments/files/17347893/cg_system.zip)

**Comment by ricalessandri** [comment]
As I understand, that script is outaded, at least for protein visualization. Try this out: https://github.com/Martini-Force-Field-Initiative/MartiniGlass
